Formula/unzip.rb

```diff
@@ -53,23 +53,25 @@ class Unzip < Formula
       patches/18-cve-2014-9913-unzip-buffer-overflow.patch
       patches/19-cve-2016-9844-zipinfo-buffer-overflow.patch
       patches/20-cve-2018-1000035-unzip-buffer-overflow.patch
+      patches/20-unzip60-alt-iconv-utf8.patch
       patches/21-fix-warning-messages-on-big-files.patch
       patches/22-cve-2019-13232-fix-bug-in-undefer-input.patch
       patches/23-cve-2019-13232-zip-bomb-with-overlapped-entries.patch
       patches/24-cve-2019-13232-do-not-raise-alert-for-misplaced-central-directory.patch
       patches/25-cve-2019-13232-fix-bug-in-uzbunzip2.patch
       patches/26-cve-2019-13232-fix-bug-in-uzinflate.patch
       patches/27-zipgrep-avoid-test-errors.patch
+      patches/28-cve-2022-0529-and-cve-2022-0530.patch
     ]
   end
 
   def install
-    args = %W[
-      CC=#{ENV.cc}
-      LOC=-DLARGE_FILE_SUPPORT
-      D_USE_BZ2=-DUSE_BZIP2
-      L_BZ2=-lbz2
-      macosx
+    args = [
+      "CC=#{ENV.cc}",
+      "LOC=-DLARGE_FILE_SUPPORT -DUNICODE_SUPPORT -DUNICODE_WCHAR -DUTF8_MAYBE_NATIVE -DNO_WORKING_ISPRINT",
+      "D_USE_BZ2=-DUSE_BZIP2",
+      "L_BZ2=-lbz2",
+      "macosx",
```

Comment:
```suggestion
    loc_macros = %w[
      -DLARGE_FILE_SUPPORT
      -DUNICODE_SUPPORT
      -DUNICODE_WCHAR
      -DUTF8_MAYBE_NATIVE
      -DNO_WORKING_ISPRINT
    ]
    args = %w[
      CC=#{ENV.cc}
      LOC=#{loc_macros.join(" ")}
      D_USE_BZ2=-DUSE_BZIP2
      L_BZ2=-lbz2
      macosx
```
Can we also add a comment on what the macros we're enabling do? Why do we want to enable them?

Reply:
> Can we also add a comment on what the macros we're enabling do? Why do we want to enable them?

This part of my last review is not yet resolved.

Reply:
Yeah, was squashing in the meanwhile. GitHub is unfortunately rather broken, and does not allow me to commit suggestions and make edits to files altogether. Should be good now.